scripts/gufi_stats

```diff
@@ -1105,6 +1105,17 @@ OTHERS = [
     ['gid-size',              gid_size],
 ]
 
+
+# a helper function to show the gufi_query being executed. Helpful for debugging and education.
+def my_pretty_print(cmd_tokens):
```

Comment:
maybe give this function another name

Reply:
Called it print_query and moved it into gufi_common